cognite/neat/explorer/explorer.py

```diff
@@ -75,44 +74,14 @@
     allow_headers=["*"],
 )
 
+# UI cache store
 cache_store = {}
 
-cdf_store = CdfStore(
-    cdf_client,
-    config.cdf_default_dataset_id,
-    workflows_storage_path=config.workflows_store_path,
-    rules_storage_path=config.rules_store_path,
-)
-
-if config.workflow_downloader_filter:
-    cdf_store.load_workflows_from_cfg_by_filter(config.workflow_downloader_filter)
-
-workflow_manager = WorkflowManager(
-    cdf_client,
-    config.workflows_store_type,
-    config.workflows_store_path,
-    config.rules_store_path,
-    config.cdf_default_dataset_id,
-)
-workflow_manager.load_workflows_from_storage_v2()
 
-triggers_manager = TriggerManager(workflow_manager=workflow_manager)
-triggers_manager.start_http_listeners(app)
-triggers_manager.start_time_schedulers()
-
-
-def signal_handler(sig, frame):
-    print("You pressed Ctrl+C!")
-    triggers_manager.stop_scheduler_main_loop()
-    exit(0)
-
-
-# signal.signal(signal.SIGINT, signal_handler)
-# signal.signal(signal.SIGTERM, signal_handler)
 @app.on_event("shutdown")
 def shutdown_event():
     logging.info("FastApi shutdown event")
-    triggers_manager.stop_scheduler_main_loop()
+    neat_app.stop()
```

Comment:
Can be replaced by a `Lifespan` event. The `shutdown` even is deprecated. https://fastapi.tiangolo.com/advanced/events/#lifespan

Reply:
Will replace